src/main/java/de/hysky/skyblocker/mixins/ClientPlayNetworkHandlerMixin.java

```diff
@@ -103,4 +104,14 @@ public abstract class ClientPlayNetworkHandlerMixin {
         }
         return entity;
     }
+
+    @Inject(method = "onEntityTrackerUpdate", at = @At("TAIL"))
+    private void skyblocker$onEntityTrackerUpdate(EntityTrackerUpdateS2CPacket packet, CallbackInfo ci, @Local Entity entity) {
+        if (!SkyblockerConfigManager.get().general.compactDamage.enabled || entity == null || entity.getType() != EntityType.ARMOR_STAND) return;
```

Comment:
You can replace the null entity and type check with `entity instanceof ArmorStandEntity entity` which combines the two and avoids the cast below